app/package.json

```diff
@@ -8,11 +8,14 @@
     "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "lint": "NEXT_TELEMETRY_DISABLED=1 next lint",
+    "lint-fix": "NEXT_TELEMETRY_DISABLED=1 next lint --fix",
     "postinstall": "mkdir -p ./public && cp -R ./node_modules/@uswds/uswds/dist/ ./public/uswds/",
     "start": "NEXT_TELEMETRY_DISABLED=1 next start",
     "storybook": "start-storybook -p 6006",
     "storybook-build": "build-storybook",
     "test": "jest --ci --coverage",
+    "test-update": "jest --update-snapshots",
```

Comment:
is this the same as `-- -u`?

Reply:
Yea, I think the verbose name is nice to use in here and readme's to make it clear what the flag is doing